app/Bus/Commands/ComponentGroup/AddComponentGroupCommand.php

```diff
@@ -40,14 +40,28 @@
     public $collapsed;
 
     /**
+     * To whom should the component group be visible?
+     *
+     * @var int
+     */
+    public $visible;
```

Comment:
Since we no longer have a third option, I think it should stay as it is now. This way it's identical to the `Incident` model.